### PR TITLE
broken link issue #9991 fixed

### LINKED ISSUE #9991: [Docs] Broken Link
### Current State
A broken link has been found  in https://docs.meshery.io/guides/configuration-management/importing-apps .

### Desired State

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by neerajmanivarnan**
**Notes for Reviewers**
This PR fixes the broken link in the page https://docs.meshery.io/guides/configuration-management/importing-apps

This PR fixes #9991 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0078f7b4139653b50462c24c7dfd55a4c33d873c at: https://65a8240bb54013059c71b0c2--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 9cba3eb75dad627a9a966f7e8a63ae25047cfa96 at: https://65a82f7649068a00a469b8c6--meshery-docs-preview.netlify.app

File: docs/pages/guides/configuration-management/importing-applications.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ permalink: guides/configuration-management/importing-apps
 category: configuration
 type: guides
 language: en
+redirect_from:
+  - /installation/quick-start.md
```
Comment (leecalcote) [inline docs/pages/guides/configuration-management/importing-applications.md:10]: Wrong format, sir.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by neerajmanivarnan** [comment]
This issue has been resolved

**Pull request closed without merging**